posts/2025-12-18_future-of-rainbowr/index.qmd

```diff
@@ -0,0 +1,106 @@
+we have opted to propose the simplest which will meet our needs, that rainbowR becomes an [unincorporated association](https://www.gov.uk/unincorporated-associations).
+
+To be an unincorporated association, rainbowR must have a constitution, 
+and this constitution must be formally adopted by rainbowR members at an Inaugural General Meeting.
+
+**We will hold this Inaugural General Meeting on Wednesday January 28th, at 4:30pm UTC, as part of our reguarly online monthly meet-up.**
+
+We have published our [proposed constitution](https://github.com/r-lgbtq/governance/blob/main/constitution.md) on our new governance repo. It is a standard consistition for a small unincorporated association: we have adapted it from [this example](https://www.resourcecentre.org.uk/information/constitutions/#sample). 
+
+We also need a way to formally define our membership. 
+We take our current membership to be anyone who has filled in our [join form](https://docs.google.com/forms/d/e/1FAIpQLSepXotTnoEDaIaflIWXCZNB7dptPqzQsCL6ha2VVHIqgAxXDA/viewform?usp=dialog) to date.[^1]
+
+[^1]: In future, it will be anyone who has had their request for membership approved. At present, we understand that as someone filling in our join form and subsequently receiving a 'welcome' e-mail from the rainbowR committee. The join form opened in June 2022. We started sending welcome e-mails in April 2024. We believe everyone who has filled in the join form to date satisfies the conditions for membership.
+
+## Electing the leadership committee
+The current leadership committee is comprised of the authors of this post.[^2]
+We all want to expand and democratise the rainbowR leadership, 
+and ensure that all members of the community who would like a role in running the it have an opportunity to put themselves forward to do so (we strongly encourage this!). 
+
+[^2]: For a year or so after the relaunch, rainbowR was run just by Ella. 
+In September 2023, Ella asked Hanne to help out, and Daniel shortly thereafter, 
+at which point we had a committee. 
+In April 2025, Ella asked Rhian to join, 
+to help with the process of transitioning rainbowR towards a sustainable future.
+
+We therefore propose that we move to annual elections of the leadship committee.
+The fundamentals of how this would work are set out in the proposed [constitution](https://github.com/r-lgbtq/governance/blob/main/constitution.md) 
+and expanded upon in our new [committee policy](https://github.com/r-lgbtq/governance/blob/main/committee_policy.md).[^3]
+
+[^3]: We have separated the committee policy from the constitution because there is a high bar for amending the constitution (it requires a General Meeting, and the approval of two-thirds majority of those present and eligible to vote), whereas the committee policy is a more nimble document, in that it can be amended at a committee meeting, in accordance with the rules of procudure for meetings, as set out in the constitution.
```

Comment:
```suggestion
and expanded upon in our new [committee policy](https://github.com/r-lgbtq/governance/blob/main/committee_policy.md).
```

I understand you're trying to be transparent but I feel like this actually raises more questions. "Why would you want to amend in a committee meeting etc. Suggest cut.